Resources/Prototypes/_NF/Research/civilianservices.yml

```diff
@@ -1,3 +1,17 @@
+# Tier 1
+
+- type: technology
+  id: AdvancedDropper
+  name: research-technology-advanced-dropper
+  icon:
+    sprite: _NF/Objects/Specific/Chemistry/dropper.rsi
+    state: dropper
+  discipline: CivilianServices
+  tier: 1
+  cost: 5000
+  recipeUnlocks:
+  - ServiceSelectiveDropper
```

Comment:
Sorry, should've responded earlier - seems reasonable to me to just put in food service.  This looks fine, but it seems like an odd tech.  I'll test it locally regardless.

https://github.com/new-frontiers-14/frontier-station-14/blob/c928f0fd6d0995920d2d496d6cd4cd5c81474d54/Resources/Prototypes/Research/civilianservices.yml#L39-L54

Reply:
I am perfectly fine with adding it to food service. It's been a long time since I did science stuff and basically nobody asks for food service items.

Reply:
I've pushed up a minimal set of changes to get the dropper in the protolathe and in the food service tech on a branch on my repo: https://github.com/whatston3/frontier-station-14/commit/ded625abf5fa40d886013ee88c0f7336f5bf0db9

Tested and working locally.  If you're happy with those changes, add 'em in, I'll retest and approve this.

Reply:
Good to retest, I appreciate you doing the testing. My device runs very hot and lags a lot when I test.